kodilla-testing2/src/test/java/com/kodilla/testing2/crudapp/CrudAppTestSuite.java

```diff
@@ -112,8 +114,26 @@ private boolean checkTaskExistInTrello(String taskName) throws InterruptedExcept
     public void shouldCreateTrelloCard() throws InterruptedException {
 
         String taskName = createCrudAppTestTask();
+        System.out.println("task name: " + taskName);
         sendTestTaskToTrello(taskName);
         //Assert.assertTrue(checkTaskExistInTrello(taskName));
+        deleteTestTask(taskName);
+    }
+
+    private void deleteTestTask(String taskName) throws InterruptedException {
+        driver.navigate().refresh();
+
+        while (!driver.findElement(By.xpath("//select[1]")).isDisplayed());
+
+        driver.findElements(By.xpath("//form[@class=\"datatable__row\"]")).stream()
+                .filter(anyForm ->
+                        anyForm.findElement(By.xpath(".//p[@class=\"datatable__field-value\"]"))
+                                .getText().equals(taskName))
+                .forEach(theForm -> {
+                    WebElement buttonDelete = theForm.findElement(By.xpath(".//button[4]"));
```

Comment:
Tu pi w linii 126 polegasz na kolejności elementu. Spróbuj się odnieść do niego poprzez id (to może wymagać zmiany w kodzie frontendowym) lub po prostu poprzez treść elementu (tu przycisku).